src/osd/eminline.h

```diff
@@ -295,10 +295,61 @@ inline uint8_t count_leading_ones(uint32_t val)
 #endif
 
 
-/*-------------------------------------------------
-    population_count_32 - return the number of
-    one bits in a 32-bit value
--------------------------------------------------*/
+//-------------------------------------------------
+//  parity_8 - check the parity of a 8-bit value
+//  and return true for odd or false for even
+//-------------------------------------------------
+
+#ifndef parity_8
+inline bool parity_8(uint8_t data)
+{
+	data ^= data >> 4;
+	data ^= data >> 2;
+	data ^= data >> 1;
+	return (data & 1) != 0;
+}
+#endif
+
```

Comment:
For platforms with a single instruction popcount, it would probably be faster to do popcount & 1.